llvm/test/Analysis/KernelInfo/enable-kernel-info/amdgpu.ll

```diff
@@ -0,0 +1,18 @@
+; Check when kernel-info is enabled in the AMD GPU target backend.
+
+; REQUIRES: amdgpu-registered-target
+
+; DEFINE: %{opt} = opt -disable-output %S/Inputs/test.ll \
+; DEFINE:              -mtriple="amdgcn-amd-amdhsa" 2>&1
+; DEFINE: %{fcheck-on} = FileCheck -match-full-lines %S/Inputs/test.ll
+; DEFINE: %{fcheck-off} = FileCheck -allow-empty -check-prefixes=NONE \
+; DEFINE:                 %S/Inputs/test.ll
+
```

Comment:
Test the bounds attributes?

Reply:
I don't know what you're asking.  This test checks when kernel-info is enabled by looking for its output, and it arbitrarily chooses `omp_target_num_teams` as the output to look for.  Do you want more bounds checking?  Do you want something else intsead?